server.go

```diff
@@ -68,6 +52,7 @@ func addItem(c *gin.Context) {
 		response := Response{Action: "SQL", Sucessful: false, Context: err.Error()}
 		c.IndentedJSON(http.StatusOK, response)
 	}
+	defer db.Close()
```

Comment:
Can you explain the purpose of this line?

Reply:
It closes the connection with the database, I call it out of precaution, it is not technically necessary, but it makes the program slightly safer. I used defer so I could place it there just so it can stay with the rest of the code similar to it.

Reply:
Thanks! I like to include articles or docs that support/explain my point if possible too. No need to do that here.